react/src/components/organisms/TabContainer/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './style.css';
+import { TabContext } from './context';
+
+class TabContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.setActiveTab = (tab, content = null) => {
+      const state = {
+        activeTab: tab
+      };
+      if (content) {
+        state.activeContent = content;
+      }
+      this.setState(state);
+    };
+    this.state = {
+      activeTab: null,
+      activeContent: null,
+      // eslint-disable-next-line react/no-unused-state
+      setActiveTab: this.setActiveTab,
+      nested: this.props.nested
+    };
+  }
+
+  static getDerivedStateFromProps(props, state) {
```

Comment:
storybook knob doesn't seem to be working, is the purpose of this function for making storybook work?

Reply:
The purpose of that is to keep this.state.nested updated if the nested prop changes, as it's only initialized in the constructor.

Reply:
ya it's initialized with a prop value, which you are not setting the state anywhere else. i'm just wondering if it's necessary to check for prop updates